src/Application.php

```diff
@@ -105,4 +100,32 @@ public function run()
 
         return $this;
```

Comment:
That would potentially change behavior.

This is verbatim what was already in the exception handler, and I'm not comfortable changing it as it could potentially break applications (double rendering, etc.).